x/feemarket/keeper/integration_test.go

```diff
@@ -262,7 +262,7 @@ var _ = Describe("Feemarket", func() {
 				})
 
 				It("should accept transactions with gasPrice >= baseFee", func() {
-					gasPrice := baseFee
+					gasPrice := baseFee.TruncateInt()
```

Comment:
Ideally we should change the gasPrice type to `LegacyDec` in this test and in the testutils types,e.g. `CosmosTxArgs`